src/apps/search/map/panels/BasePanel.tsx

```diff
@@ -220,6 +223,19 @@ const { data: { people = [] } = {}, loading: peopleLoading } = useLoader(onLoadP
         places.filter((place) => place.place_geometry)
       );
   }, [item, places, props.resolveGeometry]);
+  */
+
+  useEffect(() => {
+    if (props.resolveGeometry && item) {
+      const highRes = props.resolveGeometry(item);
+      updatePlace(highRes);
+    } else if (!_.isEmpty(places.filter((place) => place.place_geometry))) {
```

Comment:
TBH I didn't figure out what this branch does. Under what conditions will new places come in? I assume this code needs changing... instead of creating one FeatureCollection, we probably need to loop through each place and call `updatePlace` for each one separately?